docs/usage.md

```diff
@@ -35,15 +35,43 @@ three main sections, one each for configurations, images, and session templates.
 These sections may be specified in any order, and any of the sections may be
 omitted if desired.
 
-#### Creating CFS configurations
+#### Bootprep input file schema version
 
-The `configurations` section begins with a `configurations:` key.
+The `sat bootprep` input file is validated against a versioned schema
+definition. The input file should specify the version of the schema with which
+it is compatible under a `schema_version` key. For example:
 
-```screen
+```yaml
 ---
-configurations:
+schema_version: 1.0.2
 ```
 
+To get the current `sat bootprep` input file schema version understood by the
+version of `sat` installed on the system, look at the value of the `version` key
```

Comment:
This suggestion feels less accurate to what I'm trying to say. What if we just simplify it to:

```suggestion
The current `sat bootprep` input file schema version can be viewed with the
following command:
```

I'm really just trying to convey how to get the version of the schema that is currently understood by the version of `sat bootprep` installed on the system, so that the user can use this schema version in their input files. I'm also working on providing the schema version in the bootprep input files generated by `sat bootprep generate-example` in [CRAYSAT-1545](https://jira-pro.its.hpecorp.net:8443/browse/CRAYSAT-1545), which should make this easier for the end user.

Reply:
I like this change, but I think you'll also want to delete line 51, then.